src/pages/coupon-fraud/couponFraud.module.scss

```diff
@@ -0,0 +1,190 @@
+div.wrapper {
+  max-width: 660px;
+  padding: rem(32px) rem(20px) rem(24px) rem(20px);
+  display: flex;
+  flex-direction: column;
+  gap: rem(24px);
+
+  @include media('<=phoneLandscape') {
+    padding: rem(12px);
+  }
+
+  .innerWrapper {
+    border: 1px solid v('gray-box-stroke');
+    border-radius: rem(6px);
+    padding: rem(16px);
+
+    form {
+      display: flex;
+      flex-direction: column;
+      gap: rem(16px);
+    }
+  }
+}
+
+.shoe {
+  display: flex;
+  padding-bottom: rem(32px);
+  gap: rem(24px);
+  border-bottom: 1px solid v('gray-box-stroke');
+
+  @include media('<=phone') {
+    flex-direction: column;
+    gap: rem(8px);
+  }
+
+  .shoeDescription {
+    flex-grow: 1;
+
+    .shoeName {
```

Comment:
Do you really need such dep nesting? I will generate `.shoe .shoeDescription .shoeName` class as a result.

Reply:
Hmm I don't, I just find the nesting slightly helpful for orientation - I can mirror the underlying HTML structure and have an idea about parent-child relationships between elements without switching to the component. What is the disadvantage of long class names? I am new to Sass, might be missing something

Reply:
The general problem is the larger amount of CSS that increases page loading time and produces more work for the browser in terms of applying CSS rules to the page.

But to be honest, it does not make sense for small projects like ours, so the one problem that we will have is a high specificity of css selectors: we need to reproduce the whole cascade in case we want to redefine the rule.

Reply:
I see, but the rule specificity disadvantage applies mostly to styles we expect to reuse and tweak right? Not much of an issue for styles specific to one page, correct? But I guess it makes sense as a general rule to avoid unnecessary nesting. I have reduced the nesting here and will be more conservative with it in the future: https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/pull/81/commits/1082d7065c7809a4ac0379d8b029e5a9765603e5 

Thank you!